src/modules/ekf2/EKF2.cpp

```diff
@@ -31,6 +31,8 @@
  *
  ****************************************************************************/
 
+#include <algorithm>
```

Comment:
Unfortunately we don't have a full C++ standard library on the embedded side at the moment.